PROTO_tests/tests/numeric_test.py

```diff
@@ -3,22 +3,41 @@
 import arkouda as ak
 from arkouda.dtypes import npstr
 
+prob_size = 1000
```

Comment:
I would take this out, although it is good for running quick tests, it will overwrite any set value of the `prob_size`.

Reply:
Yep.  Another oversight.  Have removed it, committed, and pushed.